pandas/core/arrays/sparse/scipy_sparse.py

```diff
@@ -195,7 +195,7 @@ def coo_to_sparse_series(
     from pandas import SparseDtype
 
     try:
-        ser = Series(A.data, MultiIndex.from_arrays((A.row, A.col)))
+        ser = Series(A.data, MultiIndex.from_arrays((A.row, A.col)), copy=False)
```

Comment:
Should this keyword get exposed to the user (and with a default of copy=True) ?

I would say this is a similar case as `Series(ndarray)` where we decided to change the default to True but give the ability for power uses to specify copy=False

Reply:
The astype below creates a copy as far as I understand, so copy=False here should be good.